src/ident/AclIdent.cc

```diff
@@ -78,7 +78,7 @@ ACLIdent::match(ACLChecklist *cl)
         }
         // else fall through to ACCESS_DUNNO failure below
     } else {
-        debugs(28, DBG_IMPORTANT, HERE << "Can't start ident lookup. No client connection" );
+        debugs(28, DBG_IMPORTANT, "Can't start ident lookup. No client connection" );
```

Comment:
Agreed, but this PR automatically removes HERE, and nothing else should be required.